strongbox-data-service/src/main/java/org/carlspring/strongbox/gremlin/adapters/EntityTraversalAdaptersConfig.java

```diff
@@ -0,0 +1,11 @@
+package org.carlspring.strongbox.gremlin.adapters;
+
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ComponentScan
+public class EntityTraversalAdaptersConfig
```

Comment:
Do we need this class?

Reply:
yes, it used in DataServiceConfig and scans `EntityTraversalAdapter` implementations

Reply:
Okay, thanks for clarifying! :)